lib/ansible/modules/windows/win_pester.ps1

```diff
@@ -78,11 +79,18 @@ if($tags.count){
 }
 # Run Pester tests
 If (Test-Path -LiteralPath $path -PathType Leaf) {
+    if ($testParameters.keys.count) {
+        $Parameters.Script = @{Path = $Path ; Parameters = $testParameters }
+        $testParametersCheckModeMsg = "with $($testParameters.keys -join ',') parameters"
```

Comment:
Because this is a message that is being appended you should have a leading space before `with`. Also would be nice to have the variable follow the existing snake_case format of the module.

Reply:
The leading space here is avoided as there is a space after `$path` in the message, will remove the space there add a leading space in the message here.